src/test/java/com/qualityraven/imagetester/api/ImageTesterTest.java

```diff
@@ -14,6 +13,8 @@
 import java.util.Collections;
 import java.util.Properties;
 
+import static org.testng.Assert.*;
```

Comment:
If you're only using Assert.fail and Assert.assertEquals it's better to explicitly import just these two static methods rather than importing everything within the class. It allows for some compiler optimizations and makes name collisions less likely too (though that one isn't _really_ a problem here)

Reply:
Sounds good.